contributing/editing-docs.rst

```diff
@@ -87,7 +87,7 @@ Building/reviewing PRs via the CI
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Once a PR is open, you can build it for review using the
-:jenkinsjob:`BIOFORMATS-DEV-merge-docs` job on
+:mergecijob:`BIOFORMATS-build-docs` job on
 the Jenkins CI. Staging documentation is no longer deployed at a URL but you
 can download them as a zip for review with the correct styling by going to the
```

Comment:
Maybe while here can clean up the is/are it/them tenses in these staging sentences. :smiley: